src/mesh/snapshots/tests__uv_sphere_2_3_after_1_iteration_of_loop_subdivision.snap

```diff
@@ -1,6 +1,6 @@
 ---
-source: src/mesh_smoothing.rs
```

Comment:
This snapshot must have changed becuase the formula for sharp normals changed slightly. Could you explain why the change is necessary? Otherwise I would revert the change - the values seem almost the same, so I suspect the difference is only caused by different arithmetic order and floating point precision.

Reply:
See my comment above. If you want, I can search for some reference and see how to calculate the triangle normal properly.